protocol/src/main/java/org/opensearch/sql/protocol/response/format/ResponseFormatter.java

```diff
@@ -27,4 +27,11 @@ public interface ResponseFormatter<R> {
    */
   String format(Throwable t);
 
+  /**
+   * Getter for the content type of the response.
```

Comment:
```suggestion
   * Getter for the content type header of the response.
```

Reply:
updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/1fce3446613b66bee8476b87f8d8a5f5da7bb66e